daisy_workflows/export/export_sbom.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# SBOM export work is done in this script, regardless of linux distro
+
+usage() {
+  echo "Usage: export_sbom.sh --syft-source path_to_syft_source --sbom-path path_to_export_sbom"
+  echo "ExportFailed: incorrect SBOM script arguments"
+  exit 1
+}
+
+if [ $# -ne 4 ]; then
+  usage
+fi
+
+if [ $1 = "--syft-source" ] && [ $3 = "--sbom-path" ]; then
```

Comment:
Validate the flags first and exit early if it's not valid and leave the actual code block at the end when you've made sure it's valid. See the example down below:

```bash
if [ "$1" != "--syft-source" ] && [ "$3" != "--sbom-path" ]; then
   usage
fi

SYFT_SOURCE=$2
...
```

Reply:
fixed